pages/[username]/index.public.js

```diff
@@ -85,6 +90,13 @@ export default function Home({ contentListFound, pagination, userFound: userFoun
   }
 
   function OptionsMenu() {
+    const isSameUser = userFound?.username === user?.username;
+    const canNuke = !isSameUser && user?.features?.includes('ban:user') && !userFound?.features?.includes('nuked');
+    const canUpdate = isSameUser && user?.features?.includes('update:user');
```

Comment:
Já existe a variável `isAuthenticatedUser` com a mesma informação que `isSameUser`.

```suggestion
    const canNuke = !isAuthenticatedUser && user?.features?.includes('ban:user') && !userFound?.features?.includes('nuked');
    const canUpdate = isAuthenticatedUser && user?.features?.includes('update:user');
```